content/projects/guidelines.md

```diff
@@ -61,73 +61,55 @@ facilitate the implementation of open science practices in neuroscience and
 create more opportunities for collaborative work and information exchange within
 the community.
 
-<h2 style="font-family:'Pacifico';color:#fe97a0">
-   How will it work?
-</h2>
-
-#### Fill in the project submission form
+<h3 style="font-family:'Pacifico'; color:#fe97a0">
+   How does it work?
+</h3>
 
 In order to submit a project to the Brainhack Global 2021 Project Gallery,
 please follow these steps:
 
 1. Register with an event listed in the
    **[Brainhack Global 2021 Events](https://brainhack.org/global2021/events/)**.
 1. Create a **[GitHub](http://github.com/)** account.
-1. Use
-   [our project submission form](https://github.com/brainhackorg/global2021/issues/new?assignees=&labels=project&template=project-submission-template.md&title=)
-   and fill in the necessary/required information as requested.
-
-   1. Make sure that the information about your project is as clear and brief as
```

Comment:
A lot of the content from this page was already in the submission form and therefore not necessary.